src/features/admin/survey-results/[id]/tables/respondent-view-table.tsx

```diff
@@ -33,43 +35,52 @@ export const RespondentViewTable = () => {
-                              <Tooltip placement='bottomStart'>
-                                <Tooltip.Trigger>
-                                  <Icon icon='Info' size='extraSmall' />
-                                </Tooltip.Trigger>
-                                <Tooltip.Content>
-                                  <pre className='font-sans whitespace-pre-wrap break-words text-left w-56'>
-                                    {question.question_text}
-                                  </pre>
-                                </Tooltip.Content>
-                              </Tooltip>
-                            </div>
-                          ))}
-                        </th>
-                      </>
+                      <th scope='col' className='py-3 w-1/12'>
+                        Index
+                      </th>
+                      <th scope='col' className='py-3'>
+                        Name
+                      </th>
+                      <th scope='col' className='py-3 w-1/12'>
+                        Status
+                      </th>
+                      <th scope='col' className='py-3'>
+                        {survey_template_questions.map((question) => (
+                          <div className='flex gap-2 break-word' key={question.id}>
+                            <p>Question {question.sequence_no}</p>
+                            <Tooltip placement='bottomStart'>
+                              <Tooltip.Trigger>
+                                <Icon icon='Info' size='extraSmall' />
+                              </Tooltip.Trigger>
+                              <Tooltip.Content>
+                                <pre className='font-sans whitespace-pre-wrap break-words text-left w-56'>
+                                  {question.question_text}
+                                </pre>
+                              </Tooltip.Content>
+                            </Tooltip>
+                          </div>
+                        ))}
+                      </th>
                     </tr>
                   </thead>
                   <tbody>
-                    {survey_results.map((result) => (
+                    {survey_results.map((result, index) => (
                       <React.Fragment key={result.id}>
-                        <tr className='hover:bg-slate-100'>
-                          <td className='py-1 px-2 border-b w-1/4 text-center'>
+                        <tr className='hover:bg-slate-100 text-sm'>
+                          <td className='py-1 border-b w-1/12'>{index + 1}</td>
+                          <td className='py-1 border-b w-2/12'>
                             {result.users?.last_name}, {result.users?.first_name}
                           </td>
-                          <td className='py-1 px-2 border-b text-center w-1/6'>
+                          <td className='py-1 border-b w-1/12'>
+                            <Badge variant='green' size='small'>
```

Comment:
pls modify to show variant based on status. you can check getEvaluationResultStatusVariant